test/testmachinery/shoot/enable_rotate_certs_disable_test.go

```diff
@@ -0,0 +1,107 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package shoot
+
+import (
+	"context"
+	"time"
+
+	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
+	"github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
+	"github.com/gardener/gardener/test/framework"
+	"github.com/gardener/gardener/test/utils/access"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/api/resource"
+
+	"github.com/gardener/gardener-extension-registry-cache/pkg/apis/registry/v1alpha3"
+	"github.com/gardener/gardener-extension-registry-cache/test/common"
+)
+
+const (
+	rotateCertTestTimeout        = 60 * time.Minute
+	rotateCertTestCleanupTimeout = 20 * time.Minute
+)
+
+var _ = Describe("Registry Cache Extension Tests", Label("cache"), func() {
+	f := framework.NewShootFramework(nil)
+
+	f.Disruptive().Beta().CIt("should enable extension, rotate Certificate, disable extension", func(parentCtx context.Context) {
```

Comment:
The operation is `rotate CA`, not `rotate Certificate`.

```suggestion
	f.Disruptive().Beta().CIt("should enable extension, rotate CA, disable extension", func(parentCtx context.Context) {
```